### DISCUSSION TITLE
Visiting the BOSL2 wiki page on Github unauthenticated results in me getting IP banned and blocks me from accessing the login page

### DISCUSSION THREAD

**Discussion by AmericanPresidentJimmyCarter** [community/community#67916] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

To reproduce:

1. Be unauthenticated.
2. Go to the BOSL2 wiki page: https://github.com/BelfrySCAD/BOSL2/wiki
3. Go between various pages, especially shapes2d and shapes3d. It doesn't take long, maybe 5-6 page loads
4. Enjoy your new Github homepage: 

> Access to this site has been restricted.
> If you believe this is an error, please contact [Support](https://support.github.com/).
> [GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

edit: After about 10 minutes, the ban is automatically lifted.

**Comment by MFernflower** [comment]
Also happens with https://github.com/nophead/NopSCADlib/
